src/rosenbrock.jl

```diff
@@ -0,0 +1,89 @@
+#ODEROSENBROCK Solve stiff differential equations, Rosenbrock method
```

Comment:
These are the old solvers, right?  If so, I'd re-add them to `src/ODE.jl` where it was.